.github/workflows/publish-to-pypi.yml

```
@@ -0,0 +1,38 @@
+name: Publish to PyPI
+
+on: push
+
+jobs:
+  publish-pypi:
+    name: Publish to PyPI
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+      with:
+        # fecth all history so that versioneer works
+        fetch-depth: 0
+
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+
+    - name: Install dependencies
+      run: pip install wheel
+
+    - name: Build source distributions and wheels
+      run: python setup.py bdist_wheel
```

```suggestion
    - name: Build source and wheel distributions
      run: |
        # Change the versioneer format to "pre" so that the commit hash isn't
        # included (PyPI doesn't allow it). Can't do this permanently because
        # we rely the hash to indicate to the tests that this is a local
        # verison instead of a published version.
        sed --in-place "s/pep440/pep440-pre/g" setup.cfg
        python setup.py sdist bdist_wheel
        echo ""
        echo "Generated files:"
        ls -lh dist/
```

See if this works, inspired by https://github.com/fatiando/pooch/pull/196, specifically the lines at https://github.com/fatiando/pooch/blob/a9564b13a95cb4521dcf5938cea5adfd2851b385/.github/workflows/deploy.yml#L51-L61